recommend/policy.go

```diff
@@ -17,85 +17,16 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-func (r *SysRule) convertToKnoxRule() types.KnoxSys {
-	knoxRule := types.KnoxSys{}
-	fromSourceArr := []types.KnoxFromSource{}
-
-	for _, eachSource := range r.FromSource {
-		if eachSource != "" {
-			if strings.HasSuffix(eachSource, "/") {
-				fromSourceArr = append(fromSourceArr, types.KnoxFromSource{
-					Dir: eachSource,
-				})
-			} else {
-				fromSourceArr = append(fromSourceArr, types.KnoxFromSource{
-					Path: eachSource,
-				})
-			}
-		}
-	}
-
-	for _, path := range r.Path {
-		if strings.HasSuffix(path, "/") {
-
-			dirRule := types.KnoxMatchDirectories{
-				Dir:        path,
-				Recursive:  r.Recursive,
-				FromSource: fromSourceArr,
-				OwnerOnly:  r.OwnerOnly,
-			}
-			knoxRule.MatchDirectories = append(knoxRule.MatchDirectories, dirRule)
-		} else {
-			pathRule := types.KnoxMatchPaths{
-				Path:       path,
-				FromSource: fromSourceArr,
-				OwnerOnly:  r.OwnerOnly,
-			}
-			knoxRule.MatchPaths = append(knoxRule.MatchPaths, pathRule)
-		}
-	}
-
-	return knoxRule
-}
-
-// networkToKnoxRule function to include KubeArmor network rules
-func (r *NetRule) networkToKnoxRule() types.NetworkRule {
-	knoxNetRule := types.NetworkRule{}
-	fromSourceArr := []types.KnoxFromSource{}
-
-	if r.FromSource != "" {
-		if strings.HasSuffix(r.FromSource, "/") {
-			fromSourceArr = append(fromSourceArr, types.KnoxFromSource{
-				Dir: r.FromSource,
-			})
-		} else {
-			fromSourceArr = append(fromSourceArr, types.KnoxFromSource{
-				Path: r.FromSource,
-			})
-		}
-	}
-	for _, protocol := range r.Protocol {
-
-		protoRule := types.KnoxMatchProtocols{
-			Protocol:   protocol,
-			FromSource: fromSourceArr,
-		}
-		knoxNetRule.MatchProtocols = append(knoxNetRule.MatchProtocols, protoRule)
+func addPolicyRule(policy *types.KnoxSystemPolicy, r types.KnoxSystemSpec) {
```

Comment:
Please don't use pkg from auto-policy-discovery. Use the pkg from KuberArmor repo - https://pkg.go.dev/github.com/kubearmor/KubeArmor/pkg/KubeArmorPolicy/api/security.kubearmor.com/v1